app/assets/javascripts/hydranorth_add_to_collection.js

```diff
@@ -0,0 +1,32 @@
+  var remove_button = $("<button type=\"button\" class=\"btn btn-danger remove_collection_group\"><i class=\"icon-white glyphicon-minus\"></i><span>Remove the following community/collection group</span></button><br>");
+  $(document).on("click", ".add_collection_group", function(){
+  current = $('div[class^="community-group"]').last()
+  addNewGroup(current);
+  });
+  $(document).on("click", ".remove_collection_group", function(){
+  var current = $(this).parent()
+  console.log(current)
```

Comment:
Just talked with Natasha and she's going to help me revise the javascript here this afternoon.